src/jsx/import/result/results.jsx

```diff
@@ -8,120 +8,122 @@
-            } else if (asynchObject) {
-                const results = asynchObject;
+        // asynchronous callback: ajax 'done' promise
+        ajaxCaller(
+            (asynchObject) => {
+                // Append to DOM
+                if (asynchObject && asynchObject.error) {
+                    this.setState({ ajax_done_error: asynchObject.error });
+                } else if (asynchObject) {
+                    const results = asynchObject;
 
-                // enumerate and store response
-                this.setState(Object.assign({}, results))
-            }
-        // boolean to hide ajax spinner
-            this.setState({display_spinner: false});
-        }.bind(this),
-      // asynchronous callback: ajax 'fail' promise
-        function (asynchStatus, asynchError) {
-            if (asynchStatus) {
-                this.setState({ajax_fail_status: asynchStatus});
-                console.log('Error Status: ' + asynchStatus);
-            }
-            if (asynchError) {
-                this.setState({ajax_fail_error: asynchError});
-                console.log('Error Thrown: ' + asynchError);
-            }
-        // boolean to hide ajax spinner
-            this.setState({display_spinner: false});
-        }.bind(this),
-      // pass ajax arguments
-        ajaxArguments);
-    },
-    render: function() {
-      // local variables
+                    // enumerate and store response
+                    this.setState(Object.assign({}, results));
+                }
+                // boolean to hide ajax spinner
+                this.setState({ display_spinner: false });
+            },
+            // asynchronous callback: ajax 'fail' promise
+            (asynchStatus, asynchError) => {
+                if (asynchStatus) {
+                    this.setState({ ajax_fail_status: asynchStatus });
+                    console.log(`Error Status: ${asynchStatus}`);
+                }
+                if (asynchError) {
+                    this.setState({ ajax_fail_error: asynchError });
+                    console.log(`Error Thrown: ${asynchError}`);
+                }
+                // boolean to hide ajax spinner
+                this.setState({ display_spinner: false });
+            },
+            // pass ajax arguments
+            ajaxArguments,
+        );
+    }
+    render() {
+        // local variables
         const status = this.state.status;
         const titles = this.state.titles;
-        var spinner = this.state.display_spinner ? <Spinner /> : null;
+        const spinner = this.state.display_spinner ? <Spinner /> : null;
 
-      // polyfill 'entries'
+        // polyfill 'entries'
         if (!Object.entries) {
             entries.shim();
         }
 
-      // generate result
+        // generate result
         if (status == 0 && titles && titles.length > 0) {
-            var resultList = <ul className='result-list'>{
+            var resultList = (<ul className='result-list'>{
```

Comment:
I think content should follow the starting parentheses, on their own line.

Reply:
Also, you may need a space character, before the start of the, opening curly braces `{`.